src/test/java/racing/domain/car/CarTest.java

```diff
@@ -0,0 +1,64 @@
+package racing.domain.car;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import racing.strategy.strategy.AlwaysMovableStrategy;
+import racing.strategy.strategy.AlwaysNotMovableStrategy;
+
+import static org.assertj.core.api.Assertions.*;
+
+
+public class CarTest {
+
+    @Test
+    @DisplayName("이동한 후에는 포지션이 1 증가한다.")
+    public void move() {
+        Car car = new Car.Builder("car")
+                    .movableStrategy(new AlwaysMovableStrategy())
+                    .build();
+        int expected = car.getPosition() + 1;
```

Comment:
기대하는 값인 expected가 테스트에 의존적인거 같아요 
이 TC는 차를 1번 움직이고, 그에 대한 기대값이 1이기 때문에 

move() TC에 expected의 값을 받아와도 좋을거같아요, (ParameterizedTest를 활용하여) 

혹은 차에 대해서 위치를 비교하는 기능이기 때문에 아래와 같이 isEqualPosition 을 활용할수도 있을거같아요 
```suggestion

// 실제 작성해주신 코드랑은 맞지 않지만 예시를 위해서 축약했습니다 
assertThat(car.isEqualPosition(new Car(expected)));
```

Reply:
오 그렇군요, 이렇게하면 테스트에서 사용되었던 `getPosition()`도  없어질 수 있을 듯 합니다.
@CsvSource를 사용하여 ParameterizedTest를 진행하도록 변경하였습니다.

이제 getter는 출력을 위해 사용했던 것 외에는 사용되지 않는군요.
감사합니다!